public/app/features/explore/Explore.tsx

```diff
@@ -319,6 +338,7 @@ export class Explore extends React.PureComponent<Props, ExploreState> {
                           {showMetrics && graphResult && (
                             <ErrorBoundaryAlert>{this.renderGraphPanel(width)}</ErrorBoundaryAlert>
                           )}
+                          {showLogsVolume && <ErrorBoundaryAlert>{this.renderLogsVolume(width)}</ErrorBoundaryAlert>}
```

Comment:
I thought that visually it looks better as a separate panel and since the logs volume data is separated from logs model the panel can be rendered separately. It's also wrapped in a separate `ErrorBoundryAlert` so if logs volume crashed it wouldn't block logs panel.

Reply:
Thanks for explanation.